<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Yags</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ressource.qrc">
    <normaloff>:/ICO_Home.gif</normaloff>:/ICO_Home.gif</iconset>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks|QMainWindow::VerticalTabs</set>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>&amp;Fichier</string>
    </property>
    <addaction name="actionNouveau"/>
    <addaction name="actionOuvrir"/>
    <addaction name="actionEnregisrer"/>
    <addaction name="actionEnregistrer_sous"/>
    <addaction name="separator"/>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuEdition">
    <property name="title">
     <string>Edition</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuVue">
    <property name="title">
     <string>Vue</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAction">
    <property name="title">
     <string>Action</string>
    </property>
    <widget class="QMenu" name="menuFiltre">
     <property name="title">
      <string>Filtre</string>
     </property>
     <addaction name="actionMedian"/>
     <addaction name="actionMoyen"/>
     <addaction name="actionGaussien"/>
    </widget>
    <addaction name="menuFiltre"/>
   </widget>
   <widget class="QMenu" name="menuOption">
    <property name="title">
     <string>Option</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAffichage">
    <property name="title">
     <string>Affichage</string>
    </property>
    <addaction name="actionZoom_Avant"/>
    <addaction name="actionZoom_Arri_re"/>
   </widget>
   <widget class="QMenu" name="menuAide">
    <property name="title">
     <string>&amp;?</string>
    </property>
    <addaction name="actionAide"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionA_propos"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuEdition"/>
   <addaction name="menuVue"/>
   <addaction name="menuAction"/>
   <addaction name="menuOption"/>
   <addaction name="menuAffichage"/>
   <addaction name="menuAide"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionQuitter"/>
   <addaction name="actionOuvrir"/>
   <addaction name="actionEnregisrer"/>
   <addaction name="actionEnregistrer_sous"/>
   <addaction name="actionZoom_Avant"/>
   <addaction name="actionZoom_Arri_re"/>
  </widget>
  <action name="actionNouveau">
   <property name="text">
    <string>&amp;Nouveau</string>
   </property>
  </action>
  <action name="actionOuvrir">
   <property name="icon">
    <iconset resource="ressource.qrc">
     <normaloff>:/open.png</normaloff>:/open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Ouvrir...</string>
   </property>
   <property name="statusTip">
    <string>Open file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionEnregisrer">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="ressource.qrc">
     <normaloff>:/save.png</normaloff>:/save.png</iconset>
   </property>
   <property name="text">
    <string>Enregistrer</string>
   </property>
  </action>
  <action name="actionEnregistrer_sous">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="ressource.qrc">
     <normaloff>:/save_as.png</normaloff>:/save_as.png</iconset>
   </property>
   <property name="text">
    <string>Enregistrer sous...</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="icon">
    <iconset resource="ressource.qrc">
     <normaloff>:/quitter.png</normaloff>:/quitter.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quitter</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionMedian">
   <property name="text">
    <string>Median</string>
   </property>
  </action>
  <action name="actionMoyen">
   <property name="text">
    <string>Moyen</string>
   </property>
  </action>
  <action name="actionGaussien">
   <property name="text">
    <string>Gaussien</string>
   </property>
  </action>
  <action name="actionZoom_Avant">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="ressource.qrc">
     <normaloff>:/zoom_in.png</normaloff>:/zoom_in.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Avant</string>
   </property>
   <property name="shortcut">
    <string>+</string>
   </property>
  </action>
  <action name="actionZoom_Arri_re">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="ressource.qrc">
     <normaloff>:/zoom_out.png</normaloff>:/zoom_out.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Arri√®re</string>
   </property>
   <property name="shortcut">
    <string>-</string>
   </property>
  </action>
  <action name="actionAide">
   <property name="text">
    <string>Aide</string>
   </property>
  </action>
  <action name="actionA_propos">
   <property name="text">
    <string>A propos...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ressource.qrc"/>
 </resources>
 <connections/>
</ui>
